CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.2)
 
-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")
-
+#set(CMAKE_VERBOSE_MAKEFILE ON)
```

Comment:
Again, was there a reason to add this line? I removed it just recently, because now you can specify the additional CMake flags on the command-line (`configure.sh -DCMAKE_VERBOSE_MAKEFILE=ON`).

This and a few other changes here seem to me to revert CMakeLists.txt to an older version – I guess it's accidental?